benchmarks/flask_sqli/gunicorn.conf.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+def post_fork(server, worker):
+    # Set lower defaults for ensuring profiler collect is run
```

Comment:
Love the comments :+1:

Reply:
Not really mine, they're originally from flask_simple benchmark